snap/snapcraft.yaml

```diff
@@ -155,13 +155,23 @@ apps:
     plugs:
       - network
       - network-bind
+  vault-cli:
+    adapter: none
+    command: bin/vault
+    environment:
+      VAULT_CONFIG: "$SNAP_DATA/config/security-secret-store/vault-config.hcl"
+      VAULT_ADDR: "https://localhost:8200"
+      VAULT_SKIP_VERIFY: "true"
```

Comment:
why is this arg needed? If we need to skip verification when talking to a local vaul in the snap, I'd much rather the user have to provide an "skip-verify" arg than us always hard-coding it here

Reply:
Ping on this